src/manager.cc

```diff
@@ -385,6 +387,9 @@ bool fastcat::Manager::ConfigJSDBusFromYaml(YAML::Node node)
     } else if (0 == device_class.compare("El3318")) {
       device = std::make_shared<El3318>();
 
+    } else if (0 == device_class.compare("Ild1900")) {
+      device == std::make_shared<Ild1900>();
+
```

Comment:
- [ ] You need to add a similar entry to the offline bus to make use of the offline sensor (around line 577)

Reply:
Good to know! I missed this in the other drivers I wrote. Adding them along with this one.